### PR TITLE
Fix rust_smbclient_sys crate use

### PR DESCRIPTION
**Pull request by 0X1A**
- Crate was renamed but not renamed in wrapper code

cc @cholcombe973

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -8,3 +8,4 @@ pub mod smbc;
 pub use crate::{error::*, smbc::*};
 
 pub use crate::parser::*;
+extern crate rust_smbclient_sys;
```
Comment (cholcombe973) [inline src/lib.rs]: This shouldn't be needed because we're using the 2018 edition rust :)

File: src/smbc.rs [hunk 2]
```diff
@@ -61,6 +62,16 @@ fn is_einval<T: Eq + From<i8>>(t: T) -> IoResult<T> {
     }
 }
 
+/// Converts a pointer to the beginning of a C string and it's length and puts its
+/// equivelant value casted to u8
+fn str_ptr_to_u8(str_len: u32, str_ptr: *const i8, out_buffer: &mut Vec<u8>) {
+    for index in 0..str_len {
+        trace!(target: "smbc", "namelen : {}", str_len);
```
Comment (cholcombe973) [inline src/smbc.rs]: Are you sure you want to print this on every iteration of the loop? I feel like it should be printed once beforehand

File: src/smbc.rs [hunk 3]
```diff
@@ -61,6 +62,16 @@ fn is_einval<T: Eq + From<i8>>(t: T) -> IoResult<T> {
     }
 }
 
+/// Converts a pointer to the beginning of a C string and it's length and puts its
+/// equivelant value casted to u8
+fn str_ptr_to_u8(str_len: u32, str_ptr: *const i8, out_buffer: &mut Vec<u8>) {
```
Comment (cholcombe973) [inline src/smbc.rs]: I'm wondering if `Cstr::from_ptr()` would do the same thing as this?  I think it does without taking ownership.  Or a `CString::from_raw()` to take ownership.  Maybe I'm missing something here.

**Review by cholcombe973 — CHANGES_REQUESTED** [review]
The only real change I think that's needed is to drop the `extern crate call`.  Other than that I just have a few questions :).  Thanks for putting in this PR

File: src/smbc.rs (hunk 3 above)
Comment (0X1A) [reply src/smbc.rs]: Yeah, looking at this again we can just use CStr::from_ptr

File: src/lib.rs (hunk 1 above)
Comment (0X1A) [reply src/lib.rs]: 👍

**Review by cholcombe973 — APPROVED** [review]
Looks good!

**Pull request merged**